pkg/kubectl/cmd/config/delete_context_test.go

```diff
@@ -65,7 +65,7 @@ func (test deleteContextTest) run(t *testing.T) {
 	pathOptions.EnvVar = ""
 
 	buf := bytes.NewBuffer([]byte{})
-	cmd := NewCmdConfigDeleteContext(buf, pathOptions)
+	cmd := NewCmdConfigDeleteContext(buf, nil, pathOptions)
```

Comment:
I think it will be safer to pass in a `io.writer` instead of nil. Otherwise, it will panic if we print some error message in the stderr in the future.